components/core-debug/dbgsst15.cc

```diff
@@ -328,11 +328,15 @@ DbgSST15::tickleBits()
         //TODO watch point on size change
     }
     if (tickle_counter % 13 == 0 ) {
-        // replace the front element
+        // perturb one element but keep the size the same
```

Comment:
The unperturbed elements are rotated to front, so if you just look at before/after values, every element has changed.